cognite/neat/graph/extractors/_mock_graph_generator.py

```diff
@@ -0,0 +1,353 @@
+"""This module is used to generate mock graph data for purposes of testing of NEAT.
+It is a bit ugly and needs some proper refactoring, but it is not a priority at the moment.
+"""
```

Comment:
Suggest adding a Jira task on this in the technical debt epic.

Reply:
Abs!